### PR TITLE
Feature/94 wallet address needs to be configurable

### LINKED ISSUE #94: WalletAddress needs to be configurable
### Expected behaviour

Updating the wallet address input in the settings drawer should update the wallet address being used by the application.

### Actual behaviour

No change occurs.

### Steps to reproduce behaviour

Try to change the wallet address in the settings drawer.

### PR DESCRIPTION
**Pull request by ShaunBulbrook**
### Description

Wallet address can now be updated from the settings drawer component. 

### Issues fixed

Fixes #94 

### Checklist

- [x] `npm test` returns no warnings or errors.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -50,15 +47,16 @@ class App extends React.Component {
...
 
-				<SettingsDrawer
+				<ConnectedSettingsDrawer
 					isOpen={settingsDrawerIsOpen}
 					onClose={this.toggleSettingsDrawer}
+					heading="Settings"
+
+					trustSettingValue="1"
+					onSave={this.toggleSettingsDrawer}
 				>
```
Comment (dan1elhughes) [inline src/App.js]: We don't need `<X></X>` here as there's no child elements, this can be `<X/>`

File: src/actions/index.js [hunk 2]
```diff
@@ -29,6 +32,19 @@ export const getAllTransactions = () => dispatch => {
 		});
 };
 
+export const clearTransactions = () => {
+	return({
```
Comment (dan1elhughes) [inline src/actions/index.js]: Immediately returning functions can be collapsed when using fat arrows

File: src/actions/index.js [hunk 3]
```diff
@@ -29,6 +32,19 @@ export const getAllTransactions = () => dispatch => {
...
+		type: types.CLEAR_TRANSACTIONS,
+	});
+};
+
 export const enableTransactionPolling = () => dispatch => {
 	setInterval(() => dispatch(getAllTransactions()), 5000);
 };
+
+export const setWalletAddress = newWalletAddressValue => {
```
Comment (dan1elhughes) [inline src/actions/index.js]: Can we wrap this with `checkFSA`, and also `newWalletAddressValue` -> `walletAddress` - we know it's new and that it's a value

File: src/App.js (hunk 1 above)
Comment (ShaunBulbrook) [reply src/App.js]: Fixed

File: src/actions/index.js (hunk 2 above)
Comment (ShaunBulbrook) [reply src/actions/index.js]: Done.

File: src/actions/index.js (hunk 3 above)
Comment (ShaunBulbrook) [reply src/actions/index.js]: Changed to `updateSettings`.  - Wrapped with `checkFSA`

**Pull request merged**